Library/Homebrew/cleanup.rb

```diff
@@ -144,9 +171,10 @@ def cleanup_logs
       end
     end
 
-    def cleanup_cache
-      return unless cache.directory?
-      cache.children.each do |path|
+    def cleanup_cache(entries = nil)
```

Comment:
Maybe do `cleanup_lockfiles` and `rm_ds_store` like this too and use them in `cleanup_formula`?

Reply:
Sorry, I see the `rm_ds_store` is done in another PR. Would be good to have a PR for `cleanup_lockfiles` too.